### DISCUSSION TITLE
Foundry-devops project updated, foundry.toml change needed if you install after 20 Nov 2023

### DISCUSSION THREAD

**Discussion by rocknet** [Cyfrin/foundry-full-course-cu#1130] [General]
Hi all.  Patrick merged a PR I submitted, answering his call to action in Lesson 11 to migrate `foundry-devops` from `ffi` to Foundry Files.  The Usage section of the README has been updated with details, but if you watch the videos recently, install foundry-devops, and it doesn't work, it's because it doesn't use `ffi` anymore.  See [the README](https://github.com/Cyfrin/foundry-devops/tree/main#usage) for details.

**Comment by VoteTheWorld** [comment]
I just figured it out through ChatGPT and found you posted here🙃

**Reply by GaraninPro** [reply]
So what  does  it use  now ?  We need  fs_permissions = [{ access = "read", path = "./broadcast" }] only in tomlfile  or WHAT ?

**Reply by rocknet** [reply]
Yes.  See the README I linked above.  The usage section assumes you're starting from new, so if you're updating to the latest version, you can (and should) remove the `ffi` item from the `foundry.toml` file.

**Comment by GaraninPro** [comment]
So, I deleted ``` ffi = true ``` and got error again  but then deleted all json files  in broadcast folder. BOOM !!! Everything is fine. Try it frens  if nothing works.
